pkg/controllers/uiplugin/plugin_info_builder.go

```diff
@@ -102,6 +101,30 @@ func PluginInfoBuilder(plugin *uiv1alpha1.UIPlugin, pluginConf UIPluginsConfigur
 				},
 			}
 
+			return pluginInfo, nil
+		}
+	case uiv1alpha1.TypeTroubleshootingPanel:
+		{
```

Comment:
Maybe a better approach would be to follow [this](https://github.com/rhobs/observability-operator/pull/477/files#diff-8fe461b7b9b03ac1fddf5f028a5e339ab5fc6414582abee75fdff05fad6d77aa) pattern and extract this into its own file for clarity. We might also need to add support so the korrel8r endpoint and namespace can be configured

Reply:
I agree, the troubleshooting panel and distributed tracing have been move out into their own files. Korrel8r name and namespace support for the proxy has also been added